frontend/src/Components/HomePage/HomePage.jsx

```diff
@@ -20,7 +20,7 @@ const HomePage = ({ setAccessToken }) => {
                      onClick={() => {
                         redirectTo(navigate, ROUTES.ADMIN, 1);
                      }}
-                     className="btn btn-danger mr-3"
+                     className={userRole === "role_admin" ? `btn btn-danger mr-3` : "d-none"}
```

Comment:
Make an object or enum that describes roles and reuse it through the app. This way if role_admin changes, you would need to change everywhere. Try to keep all hardcoded stuff in one place so it is easier for possible change.

Reply:
Thanks. Fixed.